From 19fe5c4d07e48dce1a64085f96653cad4498db2e Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?V=C3=ADctor=20Manuel=20J=C3=A1quez=20Leal?=
 <vjaquez@igalia.com>
Date: Mon, 12 Feb 2018 19:00:36 +0100
Subject: [PATCH] plugins: update mesa's vendor string in whitelist

Mesa has updated its VA-API Gallium driver vendor string:

https://cgit.freedesktop.org/mesa/mesa/commit/?id=5db29d62ce1fefa3f2ee6e4a4688576fde4bde4a

This patch tries to cover both, the old and the new one.

https://bugzilla.gnome.org/show_bug.cgi?id=793386
---
 gst/vaapi/gstvaapipluginutil.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gst/vaapi/gstvaapipluginutil.c b/gst/vaapi/gstvaapipluginutil.c
index 9e733059..0ff46bc0 100644
--- a/gst/vaapi/gstvaapipluginutil.c
+++ b/gst/vaapi/gstvaapipluginutil.c
@@ -836,7 +836,7 @@ gst_vaapi_driver_is_whitelisted (GstVaapiDisplay * display)
   guint i;
   static const gchar *whitelist[] = {
     "Intel i965 driver",
-    "mesa gallium vaapi",
+    "mesa gallium",
     NULL
   };
 
-- 
2.11.1

